pkg/common/keys.go

```diff
@@ -236,6 +238,14 @@ func (rk *redisKeys) WorkspaceVolumePathDownloadToken(token string) string {
 	return fmt.Sprintf(workspaceVolumePathDownloadToken, token)
 }
 
+func (rl *redisKeys) WorkspaceAuthorizedToken(token string) string {
+	return fmt.Sprintf(workspaceAuthorizedToken, token)
+}
+
+func (rl *redisKeys) WorkspaceAuthorizedWorkspace(workspaceId string) string {
```

Comment:
Where's this being used?

Reply:
removed. I was testing a few things with hset that didnt work out